itl/plugins-contrib.d/network-services.conf

```diff
@@ -113,6 +113,10 @@ object CheckCommand "fail2ban" {
 			set_if = "$fail2ban_perfdata$"
 			description = "If set to true, activate the perfdata output"
 		}
+		"-j" = {
+			value = "$fail2ban_jail$"
+			description = "Specify the name of the specific jail to monitor; default is to omit and thus monitor all jails"
```

Comment:
I'd say "omitted by default, i.e. all jails are being monitored"

Reply:
fixed in updated commit